JS/Lesson35/onder/my-app/src/components/SearchBar/index.jsx

```diff
@@ -1,32 +1,30 @@
+import React, { useState } from 'react';
 
-import { useState } from "react";
 
-export const SearchBar = ({setRecipes}) => {
-    const [value, setValue] = useState("");
+const SearchBar = ({ setRecipes }) => {
+  const [value, setValue] = useState('');
 
-    function handleSubmit (event) {
-        event.preventDefault();
+  const handleSubmit = (event) => {
+    event.preventDefault();
 
-        fetch(`https://www.themealdb.com/api/json/v1/1/search.php?s=${value}`)
-        .then(response => response.json())
-        .then(data => {
-          // If any recipes are found, save them to the App component's recipes state
-          if (data.meals) {
-            setRecipes(data.meals);
-          // If no recipes are found, reset the App component's recipes state 
-          // Alternatively, you can remove the else clause to keep the old recipes on the screen
-          } else {
-            setRecipes([]);
-          }
-          setValue("");
-        });
-    };
+    fetch(`https://www.themealdb.com/api/json/v1/1/search.php?s=${value}`)
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.meals) {
+          setRecipes(data.meals);
+        } else {
+          setRecipes([]);
+        }
+        setValue('');
+      });
+  };
 
-    return(
-      <form onSubmit={handleSubmit}>
-          <input onChange={(event) => setValue(event.target.value)} value={value} type="text" />
-          <button type="submit">Search</button>
-      </form>
-    )
+  return (
+    <form onSubmit={handleSubmit}>
+      <input onChange={(event) => setValue(event.target.value)} value={value} type="text" />
+      <button type="submit">Search</button>
+    </form>
+  );
+};
 
-};
\ No newline at end of file
+export default SearchBar;
```

Comment:
There's a number of benefits of using named exports (`export const SearchBar = () => ....`) instead of default exports in react (`export default SearchBar;`).  
1. Named exports make it easier to understand what you're importing from a module. There's no guessing about what a file exports or having to peek inside to see the default export. This clarity is especially handy in larger projects where you're juggling loads of components.
2. With default exports, every importer can name the import whatever they like. This flexibility can lead to inconsistency across your project. For instance, one file could import Button as ButtonComponent and another as Btn. Named exports enforce consistent naming, which makes your codebase easier to understand at a glance.
3. Suppose you've got a component or function that needs a name change. If it's a default export, you'll have to manually update the name everywhere it's imported. With named exports, modern IDEs and text editors can automatically update these imports for you. Plus, if you're exporting multiple values from a single module, it's easier to add, remove, or modify exports without affecting the existing import statements in other files.
4. When it comes to optimizing your bundle size, named exports play nicer with tree shaking than default exports. Tree shaking is the process used by bundlers like Webpack to eliminate unused code from the final bundle. Named exports make it easier for these tools to determine which parts of a module are used and which aren't, leading to potentially smaller bundles.